libraries/aws_ecs_service.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'aws_backend'
+
+class AWSECSService < AwsResourceBase
+  name 'aws_ecs_service'
+  desc 'Describes the specified services running in your cluster.'
+
+  example "
+    describe aws_ecs_service(cluster: 'test-cluster', services: 'test1') do
+      it { should exist }
+    end
+  "
+  def initialize(opts = {})
+    opts = { services: opts } if opts.is_a?(String)
```

Comment:
this is singular resource why does the arg name in plural?

Reply:
Updating